### PR TITLE
Feat(format): add sklearn and xgboost model format validate

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

Feat(format): add sklearn and xgboost model format validate

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj @gaocegege 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by judgeeeeee** [comment]
单验 *.json 不容易辨别。加些规范？

**Review by gaocegege — COMMENTED** [review]
/lgtm

File: pkg/model/format.go [hunk 1]
```diff
@@ -220,3 +226,29 @@ func (f Format) validateForTensorRT(modelPath string, files []os.FileInfo) error
...
+
+func (f Format) validateForXGBoost(modelPath string, files []os.FileInfo) error {
+	var xgboostFileFlag bool
+	for _, file := range files {
+		if path.Ext(file.Name()) == ".xgboost" {
+			xgboostFileFlag = true
+		}
+	}
+	if !xgboostFileFlag {
+		return fmt.Errorf("there are no *.json file in %v directory", modelPath)
```
Comment (judgeeeeee) [inline pkg/model/format.go:251]:
```suggestion
		return fmt.Errorf("there are no *.xgboost file in %v directory", modelPath)
```

**Pull request merged**